pkg/util/pki/csr.go

```diff
@@ -193,16 +193,6 @@ func SignCertificate(template *x509.Certificate, issuerCert *x509.Certificate, p
 		return nil, nil, fmt.Errorf("error encoding certificate PEM: %s", err.Error())
 	}
 
-	// don't bundle the CA for selfsigned certificates
-	// TODO: better comparison method here? for now we can just compare pointers.
-	if issuerCert != template {
-		// bundle the CA
-		err = pem.Encode(pemBytes, &pem.Block{Type: "CERTIFICATE", Bytes: issuerCert.Raw})
-		if err != nil {
-			return nil, nil, fmt.Errorf("error encoding issuer cetificate PEM: %s", err.Error())
-		}
-	}
```

Comment:
How come we've removed this snippet of code? I think I sort of understand, but not certain 🤔 @mikebryant

Reply:
This is now being handled in `pkg/issuer/ca/issue.go` by the line `certPem = append(certPem, chainPem)`

This code path is only used by the `selfsigned` and `ca` issuer types, and the `selfsigned` one would also ignore this condition, so I think it's safe to move it to the `ca` issuer code